chromium_src/chrome/browser/renderer_context_menu/render_view_context_menu.cc

```diff
@@ -755,6 +780,20 @@ void BraveRenderViewContextMenu::InitMenu() {
         index.value() + 1, IDC_CONTENT_CONTEXT_OPENLINK_SPLIT_VIEW,
         IDS_CONTENT_CONTEXT_SPLIT_VIEW);
   }
+
+  std::optional<size_t> print_index =
+      menu_model_.GetIndexOfCommandId(IDC_PRINT);
+  if (print_index.has_value()) {
+    adblock_submenu_model_.AddItemWithStringId(
+        IDC_ADBLOCK_CONTEXT_BLOCK_ELEMENT, IDS_ADBLOCK_CONTEXT_BLOCK_ELEMENT);
+    adblock_submenu_model_.AddItemWithStringId(
+        IDC_ADBLOCK_CONTEXT_MANAGE_CUSTOM_FILTERS,
+        IDS_ADBLOCK_CONTEXT_MANAGE_CUSTOM_FILTERS);
+
+    menu_model_.InsertSubMenuWithStringIdAt(
+        *print_index + 1, IDC_ADBLOCK_CONTEXT_TOOLS, IDS_ADBLOCK_CONTEXT_TOOLS,
+        &adblock_submenu_model_);
```

Comment:
Have you tried [`SetIcon`](https://source.chromium.org/chromium/chromium/src/+/main:ui/base/models/simple_menu_model.h;l=216?ss=chromium) and/or [`SetMinorIcon`](https://source.chromium.org/chromium/chromium/src/+/main:ui/base/models/simple_menu_model.h;l=225?ss=chromium) here? I assume one of those should work if we want to restore the Brave logo